test/test_with_xpress.jl

```diff
@@ -551,9 +551,17 @@ end
     end
 
     @testset "Lookback Demand Charges" begin
+        data = JSON.parsefile("./scenarios/lookback_rate.json")
```

Comment:
it would be good to add a test where remove_tiers is used, but this isn't a high priority, so maybe just add an issue for now